srcpkgs/triggerhappy/template

```diff
@@ -0,0 +1,17 @@
+# Template file for 'triggerhappy'
+pkgname=triggerhappy
+version=0.5.0
+revision=1
+wrksrc="${pkgname}-release-${version}"
+build_style=gnu-makefile
+hostmakedepends="pkg-config perl"
+short_desc="Lightweight hotkey daemon"
+maintainer="Evgeni Dobrev <evgeni@studio-punkt.com>"
+license="GPL-3.0-or-later"
+homepage="https://github.com/wertarbyte/triggerhappy"
+distfiles="https://github.com/wertarbyte/triggerhappy/archive/release/${version}.tar.gz"
+checksum=af0fc196202f2d35153be401769a9ad9107b5b6387146cfa8895ae9cafad631c
+
+post_extract() {
+	sed -i 's/sbin/bin/g' Makefile
```

Comment:
please replace with `vsed`

Reply:
Thank you for the review.
I see multiple possibilities to change the destination folder in the Makefile for this particular package:
- use vsed and modify the Makefile in post_extract 
- add a patch under the "patches" folder to modify the Makefile at the patch step
- use vinstall and install the executables without calling the "install" target

What would be the preferable solution?

Reply:
vsed is fine for small changes like this. It will only break if the pattern no longer exists in the file and vsed ensures that you will be notified about that breakage. For bigger changes patches are prefereable. Vinstall should only be used when the package does not offer an installation mechanism or it is completely broken.

Reply:
Thanks you for the feedback. I will correct accordingly.